substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfSections.java

```diff
@@ -0,0 +1,2344 @@
+/*
```

Comment:
This file would profit from getting partitioned into multiple source files.

Reply:
All the section implementation classes are now in separat efiles and they are no longer inner classes of class SwarfSections